### PR TITLE
feat: exclude archived items from list

### PR DESCRIPTION
**Pull request by jacobsimeon**
- Update graphql schema
- Query for `isArchived` property of SavedItem
- Include `isArchived: false` filter when fetching data from API
- Persist `isArchived` property in core data
- Add `isArchived = false` predicate to Item fetch request

### CODE REVIEW

**Review by dskuza — APPROVED** [review]

File: PocketKit/Sources/Sync/Requests.swift [hunk 1]
```diff
@@ -7,12 +7,20 @@ import CoreData
 
 public enum Requests {
     public static func fetchItems() -> NSFetchRequest<Item> {
+        let request = fetchAllItems()
```
Comment (dskuza) [inline PocketKit/Sources/Sync/Requests.swift:10]: I think the word `all` may be overloaded. `All` items may be interpreted as all items a user has, including unarchived, archived, and deleted, but here, `all` means "unarchived and not deleted". What do you think about renaming this? `fetchActiveItems`, `fetchCurrentItems`, `fetchListItems`, `fetchSomeOtherWordItems`?

**Review by dskuza — COMMENTED** [review]
Simple changes, I like it. I just have one question regarding the naming of a function!

File: PocketKit/Sources/Sync/Requests.swift (hunk 1 above)
Comment (jacobsimeon) [reply PocketKit/Sources/Sync/Requests.swift:10]: well... `All` here actually means "anything in the table". the archived/deleted flags only come into account in `fetchItems` request (which adds predicates to `fetchAllItems`)

Comment (dskuza) [reply PocketKit/Sources/Sync/Requests.swift:10]: Ah! You're right. My interpretation of this was incorrect :)

**Review by dskuza — APPROVED** [review]

**Pull request merged**